index.html

```diff
@@ -5,523 +5,1137 @@
+            class="brands-item-img"
+          />
+        </li>
+        <li class="brands-item">
+          <img
+            src="./img/logo/logo-partner2.png"
+            alt="Brand logo"
+            width="165"
+            height="40"
+            class="brands-item-img"
+          />
+        </li>
+        <li class="brands-item">
+          <img
+            src="./img/logo/logo-partner3.png"
+            alt="Brand logo"
+            width="165"
+            height="40"
+            class="brands-item-img"
+          />
+        </li>
+        <li class="brands-item">
+          <img
+            src="./img/logo/logo-partner4.png"
+            alt="Brand logo"
+            width="165"
+            height="40"
+            class="brands-item-img"
+          />
+        </li>
+        <li class="brands-item">
+          <img
+            src="./img/logo/logo-partner5.png"
+            alt="Brand logo"
+            width="165"
+            height="40"
+            class="brands-item-img"
+          />
+        </li>
+        <li class="brands-item">
+          <img
+            src="./img/logo/logo-partner6.png"
+            alt="Brand logo"
+            width="165"
+            height="40"
+            class="brands-item-img"
+          />
+        </li>
+      </ul>
 
       <!-- FAQs section -->
-      <section class="faqs">
-        <h2 class="section-name">F.A.Q</h2>
-        <h3 class="section-title">Answers to Your Questions</h3>
-        <button type="submit">Ask Us A Question</button>
-
-        <details>
-          <summary class="faqs-name">Can I request a refund?</summary>
-          <!-- answer text needed -->
-          <p></p>
-        </details>
-
-        <details open>
-          <summary>How do I access my purchased products?</summary>
-          <p>
-            Accessing your purchased products is seamless and hassle-free on our
-            digital design assets marketplace. Simply log in to your account,
-            navigate to the 'My Purchases' section, and you'll find all your
-            purchased products ready for download.
-          </p>
-        </details>
-
-        <details>
-          <summary>Are there any discounts for bulk purchases?</summary>
-          <!-- answer text needed -->
-          <p></p>
-        </details>
-
-        <details>
-          <summary>What payment methods do you accept here?</summary>
-          <!-- answer text needed -->
-          <p></p>
-        </details>
-
-        <details>
-          <summary>Can I sell my digital products on this marketplace?</summary>
-          <!-- answer text needed -->
-          <p></p>
-        </details>
+      <section class="faqs container">
+        <div class="faqs-wrapper">
+          <div class="faqs-content">
+            <h2 class="section-name franklin-normal-16">F.A.Q</h2>
+            <h3 class="section-title">Answers to Your Questions</h3>
+          </div>
+          <button type="submit" class="btn-faqs franklin-bold-16">
```

Comment:
The button with type submit  only works inside the form

Reply:
#Fixed